molgenis-one-click-importer/src/main/frontend/src/components/OneClickImporter.vue

```diff
@@ -22,16 +52,78 @@
     name: 'one-click-importer',
     data () {
       return {
-        file: null
+        options: {
+          url: '/plugin/one-click-importer/upload',
+          paramName: 'file',
+          // FIXME: more than one file now produces error, should clear for next
+          maxFiles: {
+            limit: 1,
+            message: 'You can only upload one file'
+          },
+          // This has to be a comma separated string, don't ask me why
+          acceptedFiles: 'application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
+        }
       }
     },
     methods: {
       setFile: function (event) {
         this.file = event.target.files[0]
       },
-      importFile: function () {
-        this.$store.dispatch(IMPORT, this.file)
+      importFile: function (file) {
+        console.log(file.name)
+        this.$store.dispatch(IMPORT, file)
       }
     }
   }
 </script>
+<style scoped>
+  body {
+    display: flex;
+    background: #e8e8e8;
```

Comment:
we need to use bootstrap classes or variables to be able to style the components using some sort of theme picker.